web/src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -15,25 +15,25 @@ import {
   rangeSliderConfig
 } from 'features/fireWeather/components/graphs/plotlyHelper'
 
-const observedTempColor = '#fb0058'
-const observedRHColor = '#005f87'
-const observedDewpointColor = '#02fa44'
-const forecastTempColor = '#a50b41'
-const forecastRHColor = '#17c4c4'
-const hrdpsTempColor = '#a017c2'
-const hrdpsRHColor = '#3ac417'
+const observedTempColor = '#4291FF'
+const observedRHColor = '#EB5757'
+const observedDewpointColor = '#949494'
+const forecastTempColor = '#1200DE'
+const forecastRHColor = '#910000'
+const hrdpsTempColor = '#B86BFF'
+const hrdpsRHColor = '#229B56'
 const hrdpsTempPlumeColor = 'rgba(203, 169, 214, 0.4)'
 const hrdpsRHPlumeColor = 'rgba(181, 240, 165, 0.4)'
-const rdpsTempColor = '#ea6d0e'
-const rdpsRHColor = '#026200'
+const rdpsTempColor = '#FF64DD'
+const rdpsRHColor = '#285777'
 const rdpsTempPlumeColor = 'rgba(244, 143, 65, 0.4)'
 const rdpsRHPlumeColor = 'rgba(42, 137, 137, 0.4)'
-const gdpsTempColor = '#f56c9c'
-const gdpsRHColor = '#32e7e7'
+const gdpsTempColor = '#7556CA'
+const gdpsRHColor = '#F2994A'
 const gdpsTempPlumeColor = 'rgba(255, 150, 169, 0.4)'
 const gdpsRHPlumeColor = 'rgba(148, 255, 235, 0.4)'
```

Comment:
the plume colours should be updated too - opacity should be 0.2 instead of 0.4. Just to make sure, @conbrad when we talk about "plumes" do you know what we're talking about?

Reply:
Sorry yeah wasn't sure on the figma design if there were plume color changes or not. Just cleared it up with Ambrose. The plume is the area under the lines correct?

Reply:
the plume is the lightly shaded areas for each of the weather variables. They indicate the range that each weather variable was forecasted at some point in time - e.g., because a model is run multiple times to predict wx for a given day, we should the range of values that were predicted, up to the 90th percentile. Hope that makes sense - we can always have a quick chat on Teams if that's still confusing